ClientApp/src/Pages/SubmitSighting/SubmitSighting.tsx

```diff
@@ -68,46 +75,55 @@ export function SubmitSightingForm(): JSX.Element {
             onChange={(event) => setQuantity(event.target.value)}
           />
         </label>
-
+        </div>
+        </div>
         <label className="form-label">
           Location
           <input
             className="form-input"
             value={location}
+            placeholder="Tofino, BC"
             onChange={(event) => setLocation(event.target.value)}
           />
         </label>
-
-      <label className="form-label">
+        <div className="row g-3">
+        <div className="col">
+        <label className="form-label">
         Latitude
         <input
           className="form-input"
           value={latitude}
           onChange={(event) => setLatitude(event.target.value)}
+          placeholder="49.146862"
```

Comment:
It's a slightly odd placeholder value to have, I think I'd prefer to have nothing unless we could get their longitude and latitude from their current location